### DISCUSSION TITLE
Project announcement

### DISCUSSION THREAD

**Discussion by Haarolean** [kafbat/kafka-ui#23] [Announcements]
Hi, Kafkers! Long time no see?  
  
## Preamble  
  
As you might've noticed, there hasn't been a single commit in the repo for the past six months. This all started with [this announcement](https://github.com/provectus/kafka-ui/discussions/4255). Despite the stated obligations by Provectus, no major issues (such as [CVE-2023-52251](https://nvd.nist.gov/vuln/detail/CVE-2023-52251)) have been addressed.  
  
After exploring all viable solutions and realizing our options were exhausted, we decided to fork the repo. In January, we regrouped and began working diligently on the next release, which we're excited to present to you today! We'll delve into the release details shortly.  
  
## About us  
  
We are **Kafbat**, now an independent team continuing the development of the UI you're already familiar with. Our team consists of key contributors from the original project team, who bring a wealth of experience and insight, as well as new members — our friends and colleagues — who contribute fresh ideas to meet modern demands.  
  
Our commitment is to the continuous evolution of the project, into which we've poured our hearts and souls over the years, while maintaining the foundational vision.  
*We extend our gratitude to Provectus for their past support in groundbreaking work, which serves as a cornerstone for our ongoing innovation and dedication.*  
  
## About the release  
  
- Believing in the maturity of our project, we're starting our first release from version v1.0, instead of continuing from v0.8;  
- We've made efforts to maintain backward compatibility, so updating the image name (and the helm charts repo link, if used) should suffice;  
- We encourage you to report any issues encountered in our new repo;  
- Please spread the word about our new release to your friends and colleagues! We'd appreciate your support through stars on GitHub, upvotes on HackerNews/Reddit/ProductHunt, and more!  
  
I've omitted the dev changelog here, as the announcement itself is already long enough, but here are a few major changes:  
  
- Resolved all issues with messages (filters, pagination, etc.);  
- Replaced Groovy filters with CEL, eliminating the possibility of RCEs while improving functionality;  
- ACLs are now editable, allowing for creation, editing, and deletion;  
- Numerous other fixes and enhancements.  

## Sponsorship  
  
As mentioned, we're now an independent team without external funding. This means we work on the project in our spare time. Although we'd love to dedicate more time to it, bills won't pay themselves. Your support is crucial for us from this point forward, which is why we've launched a sponsorship  program on GitHub. Here are some of the perks you can enjoy by supporting us:  
  
- A mention of your support in your profile, the project's README, and release notes, plus a custom Discord role;  
- Priority handling of your bug reports and exclusive access to a private sponsors' channel on Discord, where we offer personalized support and insights.  
  
There are also one-time sponsorship tiers available:  
  
- You can sponsor a specific bug/issue for dedicated attention from our team;  
- We offer assistance with your infrastructure;  
- We can develop custom Kafka solutions for you.  
  
More information about the tiers is available [here](https://github.com/sponsors/kafbat). Organizations can also subscribe on GitHub—feel free to encourage your companies to support us!  
### Links  
  
Release notes: https://github.com/kafbat/kafka-ui/releases/tag/v1.0.0  
Repo: https://github.com/kafbat/kafka-ui (Please leave a star!)  
Org: https://github.com/kafbat (Feel free to follow us!)  
Sponsoring: https://github.com/sponsors/kafbat  
Docs: https://ui.docs.kafbat.io  
Charts: https://ui.charts.kafbat.io  
Random capybara pic: https://postimg.cc/75vXF018 (Leave a "K" reaction if you scrolled this far)

**Comment by yermulnik** [comment]
Great to hear the development has revived 🥳 

Along with the refreshment, are there any plans to replicate AWS Public ECR repo for Kafka-UI going forward? I assume that https://gallery.ecr.aws/provectus/kafka-ui-custom-build isn't going to receive container image updates from this new project, right?

**Reply by Haarolean** [reply]
Hi, yes, we plan to ship an AMI image. It's currently under a review :)

**Reply by yermulnik** [reply]
Well, it's not about an AMI image, but rather about pushing Docker images into AWS Public ECR (which helps people in AWS to not struggle with GHCR rate limits).

**Reply by Haarolean** [reply]
Ah, I see what you mean. These images were mostly for internal use (these are custom branch deployments) and none of these contain neither release nor main branch builds.

Could you elaborate please, have you used any of these indeed?

**Reply by yermulnik** [reply]
Yep, we've been using them to deploy Kafka-UI. No issues so far.

**Reply by yermulnik** [reply]
Though it would definitely be great to have them built and pushed from main branch.

ps: the AMI approach would also work for us I guess (would require changes to code and deployment process though), while having both (AMIs and Container Images in AWS Public ECR) would be even better obviously 😺

**Reply by Haarolean** [reply]
I see. These are custom half-baked builds to test particular things out, they're not intended for production (or even general development) use 😅
We currently publish to github (and mirror to DockerHub), but I'll raise an issue and discuss the possibility of uploading the images to a public ECR as well.

**Reply by yermulnik** [reply]
> I'll raise an issue and discuss the possibility of uploading the images to a public ECR as well.

Appreciate that 👍🏻
[AWS Public ECR](https://gallery.ecr.aws/) is pretty much a salvation from GHCR and DockerHub rate limits.
Unfortunately this requires and active AWS Account (as far as I understood): https://docs.aws.amazon.com/AmazonECR/latest/public/public-getting-started.html

**Reply by yermulnik** [reply]
> I'll raise an issue and discuss the possibility of uploading the images to a public ECR as well.

@Haarolean Has there been any outcome yet?

**Reply by Haarolean** [reply]
@yermulnik #242

**Comment by mmasipgu** [comment]
Amazing!!!!!!! 

Great job and thanks from the Kafka community.

We need people like yours in that world!

Thanks again team!

**Comment by akamensky** [comment]
Thank you for keeping the project rolling! <3 

PS: Would it make sense to link announcement from provectus repo to this project? I've seen the links at the bottom comments, but not in the main post.
